### DISCUSSION TITLE
Microdialysis concentrations

### DISCUSSION THREAD

**Discussion by Vichelfer** [Open-Systems-Pharmacology/Forum#790] [Questions & Problems]
Hello!

I'm currently working to construct a model with literature human data and I found a study that has quantification of interstitial unbound concentrations in muscle and subcutaneous tissue in healthy volunteers using the microdialysis technique. 
They found an AUC tissue unbound/AUC plasma unbound ratio of 0.5 after a single dose for both tissues and 0.66 and 0.75 after 3 doses (q12h) for muscle and subcutaneous tissue respectively (the half-life of the drug is 2.3 h, and it is administered through a 1-hour infusion as a prodrug that is rapidly converted in plasma to the drug). 
In the concentration vs time profile, the initial concentrations are lower in the tissues than those in the plasma unbound, and then at the time of 4 hours, the concentrations in plasma and tissue become equal. 

I have a question about trying to simulate those microdialysis concentrations in PK-Sim. Have anyone done? 
Because I considered as interstitial unbound, but in PK-Sim these simulated concentrations are equal to that on plasma unbound, right?
Also, I thought about inserting some efflux transporter to explain those finds, but I started to wonder where these transporters are located in PK-Sim? They influence also the concentrations in the interstitial fluid or only in the tissue (meaning, only when entering the tissue cells that has an effect)?

Additionally, I thought about changing the partition coefficient, but, I was confused about the options that we have in PK-Sim: interstitial : plasma or intracellular : plasma, which one I could choose to alter in this case?

**Comment by tobiasK2001** [comment]
Dear Vichelfer,

I am not sure what you mean by this:
> Because I considered as interstitial unbound, but in PK-Sim these simulated concentrations are equal to that on plasma unbound, right?
Is it for you compound the case? Obviously this is compound dependent. E.g. for Midazolam it can be different
![image](https://user-images.githubusercontent.com/25585248/99958846-57ff0680-2d8a-11eb-9dc6-575e4a649bd6.png)

But the difference is not big. Overall I am not sure how well the PK-Sim observer describes microdialysis data, or how comparable this data is at all. I used this data only once in one project and there it was highly dependent on experienta set up (e.g. the exact position of the needle, buffer flow) and we saw high interindividual variability (e.g. fat tissue content in subcutaneous).  

Probably other may have more experience with this.

Best, Tobias

**Comment by StephanSchaller** [comment]
Dear @Vichelfer ,

as mentioned by @tobiasK2001 , this is compound-dependent.

What size is your molecule?

What you describe and observe depends on the plasma:interstitial exchange which depends on partitioning and permeability.
As you observe a lower difference in plasma:interstitial at a later point in time, partitioning is likely not much smaller than 1. But the time it takes to equilibrate indicates slow extravasation of the drug.
Thus you will need to reduce endothelial permeability:
 
![image](https://user-images.githubusercontent.com/25477643/99965360-0c9e2580-2d95-11eb-8f57-4bcf02521f5a.png)

This is reasonable for large molecules or molecules with low lipophilicity.
Get the values down to 1e-3 cm/min and you should certainly see some delay in interstitial PK.

Hope this helps,
Stephan

**Comment by Vichelfer** [comment]
Thank you so much for giving your time to respond to me!

Dear @tobiasK2001, I'm sorry for not being clear! What I tried to say was that the PK-Sim consider the passive diffusion from plasma unbound to interstitial space, and then the transport to the intracellular space from the interstitial (that, in this case, can suffer the influence from the transporters). Like in this image from the article of Willmann et al (2003):

![image](https://user-images.githubusercontent.com/67694797/100098372-b84e8080-2e3c-11eb-8cf7-4aaecf2cbbcf.png)

But now is much more clear for me, it is highly dependent on the molecule! Thank you! I'm still studying to understand more and more every day!
 
Thank you for sharing your experience! I agree that this type of data commonly presents high variability, but I think that the results can be reliable in this case (I'm hoping)!

Dear @StephanSchaller, thank you for this suggestion! 
I forgot to mention that the molecule that I'm working on is a small one. I don't know what can be considered low lipophilicity?

**Comment by StephanSchaller** [comment]
> I don't know what can be considered low lipophilicity?

Negative values or maybe values < 1 in the LogP.

What do you consider a "small" molecule? What is the molecular weight?

And what is the level of protein binding (what is the fraction unbound)?

**Comment by Vichelfer** [comment]
> Negative values or maybe values < 1 in the LogP.

I don't have experimental data of LogP but after optimizing I ended up having a value of 0.46. So I guess it could be considered low lipophilicity. Thank you!

> What do you consider a "small" molecule? What is the molecular weight?

I consider as not being a protein! The molecular weight is 604.7 g/mol. 

> And what is the level of protein binding (what is the fraction unbound)?

The fraction unbound is 80%.

**Comment by lauraolivo11** [comment]
Hello, 

I want to reopen the discussion regarding the prediction of unbound interstitial concentrations using microdialysis in human tissue. I am now working on a PBPK model to predict the unbound fraction in the interstitial compartment. The reference I am using is an article by Romain Garreau from 2022 (https://doi.org/10.1007/s40262-022-01168-5), in which they optimize the endothelial permeability for the tissue they have data available. However, I have some concerns about doing that based on optimization, and I am having trouble finding experimental data for diffusion speed in tissue. Based on the value of experimental Kp (for humans or rodents), can I input any formula for predicting endothelial permeability more reliably? Or can I just optimize the Kp for this?

Sorry for reopening this discussion after some years but we still have limited studies on this field. 

Laura
